controllers/jaegertracing/jaeger_controller.go

```diff
@@ -42,7 +42,9 @@ type JaegerReconciler struct {
 
 // +kubebuilder:rbac:groups=jaegertracing.io,resources=jaegers,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=jaegertracing.io,resources=jaegers/status,verbs=get;update;patch
-// +kubebuilder:rbac:groups=opentelemetry.io,resources=opentelemetrycollectors,verbs=get;list;watch;create;update
+// +kubebuilder:rbac:groups=jaegertracing.io,resources=jaegers/finalizers,verbs=get;list;watch;create;update;patch
```

Comment:
Why did we need only get/patch/update for otel-operator, but all verbs for this?

https://github.com/open-telemetry/opentelemetry-operator/pull/212/files

Reply:
I was testing what are the necessary permissions and I left this here, the correct way is in the PR you pointed out.  WIll change it in this PR.